### ISSUE TITLE
Inaccurate & Outdated dataset for training requests & conversational levels. 

### ISSUE THREAD

**Issue by jaysmtsnmt** [jaysmtsnmt/Python#6]
- Outdated Test set
- Outdated & limited training set
- Training set for now only contains instructions, not requests/questions. 
- There should be a removal of common words! 
- Update training algorithm, and make it independent again from self_train()
